cpp/oneapi/dal/table/test/heterogen.cpp

```diff
@@ -14,10 +14,15 @@
 * limitations under the License.
 *******************************************************************************/
 
+#include "oneapi/dal/detail/debug.hpp"
```

Comment:
I guess this can be removed if it was for printing arrays (if used for functionality can ignore this comment). I also see it included heterogen_kernels.cpp
```suggestion
```

Reply:
This is the only feedback I have, otherwise looks good to me